### PR TITLE
Adding my details to the team page

### PR DESCRIPTION
**Pull request by palmerSL**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -87,6 +87,13 @@ people:
   pic: brennan_brouillette
   position: Software & DevOps, C3G McGill
   team: implementation
+  
+  - name: Samantha Palmer
+  lastname: Palmer
+  firstname: Samantha
+  pic: Samantha_edited
+  position: Health Data Policy, UHN
+  team: implementation
```
Comment (amanjeev) [inline _data/team.yml]: @palmerSL  woot! this is great!!!!! :+1: 


one minor thing - in this weird data format spacing means nested levels.  so the lines 92 to 96 need to be at the same level as say like 89.  

If you want I can do that for you.

Comment (palmerSL) [reply _data/team.yml]: Yes, please! Thanks again for your helpful step-by-step which got me this far!

Comment (amanjeev) [reply _data/team.yml]: My bad it was not the lines 92-96  it was merely line 91 :laughing: that needed to be less indented.  So even a quicker fix.

**Comment by ljdursi** [comment]
Looks good!

**Pull request merged**